draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -77,33 +77,56 @@ This document uses the line folding strategies described in [FOLDING].
 ## Use-As-Dictionary
 
 When responding to a HTTP Request, a server can advertise that the response can
-be used as a dictionary for future requests for URLs that match the pattern
+be used as a dictionary for future requests for URLs that match the rules
 specified in the Use-As-Dictionary response header.
 
 The Use-As-Dictionary response header is a Structured Field
-{{STRUCTURED-FIELDS}} sf-dictionary with values for "match", "ttl", "type" and
-"hashes".
+{{STRUCTURED-FIELDS}} sf-dictionary with values for "match-path",
+"match-search", "match-dest", "ttl", "type" and "hashes".
 
-### match
+### match-path
 
-The "match" value of the Use-As-Dictionary header is a sf-string value that
-provides an URL-matching pattern for requests where the dictionary can be used.
+The "match-path" value of the Use-As-Dictionary header is a sf-string value
+that provides the "pathname" of a URLPattern
+(https://wicg.github.io/urlpattern/#dictdef-urlpatterninit).
```

Comment:
Citing a WICG spec gives me the willies.  A normative reference to a CG draft is sketchy, but WICG is in a class of its own.

It might be worth talking to @wanderview about whether this has any hope of turning into a REC, hopefully through a working group.  It might be possible to get this particular piece of work done in the IETF, which might be easier than trying to find a home for it in W3C-land.

Reply:
Oh yeah, I don't disagree: https://github.com/WICG/urlpattern/issues/180

It's absolutely on-path to move out of wicg and into a published spec - currently somewhere in w3c or whatwg though. I'll see if I can get a committment on exactly where it is destined to land and the timelines. @wanderview has moved on to other projects but there is a team within Chrome that has taken ownership and is going to be doing the work to complete standardization.

In the meantime, I added an editor's note to the top of the PR calling out the fact that this PR depends on a draft spec and that this ID can't be ratified until that spec is complete.

Reply:
Looks like [this](https://github.com/whatwg/sg/issues/215) is the whatwh issue tracking the move (and consensus) from wicg to whatwg.

Reply:
URLPattern is a whatwg standard now (https://urlpattern.spec.whatwg.org/) so I updated the references to point to the new doc.